async-http-client-backend/zio/src/main/scala/com/softwaremill/sttp/asynchttpclient/zio/AsyncHttpClientZioBackend.scala

```diff
@@ -13,29 +15,43 @@ import org.asynchttpclient.{
   DefaultAsyncHttpClientConfig
 }
 import org.reactivestreams.Publisher
+import io.netty.buffer.{ByteBuf, Unpooled}
 import scalaz.zio._
+import scalaz.zio.stream._
+import scalaz.zio.interop.reactiveStreams._
 
 class AsyncHttpClientZioBackend private (asyncHttpClient: AsyncHttpClient, closeClient: Boolean)
-    extends AsyncHttpClientBackend[IO[Throwable, ?], Nothing](asyncHttpClient, IOMonadAsyncError, closeClient) {
+    extends AsyncHttpClientBackend[Task, Stream[Throwable, ByteBuffer]](asyncHttpClient, IOMonadAsyncError, closeClient) {
 
-  override protected def streamBodyToPublisher(s: Nothing): Publisher[ByteBuf] =
-    s // nothing is everything
+  val runtime = new DefaultRuntime {}
```

Comment:
How do you feel about injecting the `runtime`? A user of this code will need a runtime anyway, so we do not need to start another one.